web/src/App.test.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
```

Comment:
i'd reccomend using react-testing-library as it handles the latest version of react 16.8

Reply:
sure! will do in later work as well.

Reply:
updated